plugins/rapid7_insightidr/plugin.spec.yaml

```
@@ -25,6 +25,85 @@ sdk:
   type: full
   version: 5
   user: nobody
+version_history:
+- 10.1.0 - New Trigger - `Get New Alerts`
+- 10.0.0 - Actions - `List Investigations` Sort options updated | `Get Investigation`,
+  `List Investigations`, `Create Investigation`, `Update Investigation` `Set Priority
+  of Investigation`, `Set Disposition of Investigation`, `Set Status of Investigation
+  Action`, `Assign User to Investigation`, `Seach Investigations`, `Get a Log`, `Get
+  All Logs`, `Search Alerts` output now includes additional fields
+- 9.0.0 - Actions - `Advanced Query On Log` - Now allows for either log id or log
+  name to be used
+- 8.2.0 - Actions -  `Advanced Query On Log Set` and `Advanced Query On Log` - optimized
+  data fetching mechanisms
+- 8.1.1 - Extended error logging for all the actions
```

Also I noticed you changed these to `-` from `:`. 
If you wrap it in quotes like this you can put the `:` back as normal
```suggestion
- "8.1.1 - dadadada: Extended error logging for all the actions"
```